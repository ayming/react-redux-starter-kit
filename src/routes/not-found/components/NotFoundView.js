import React, { PureComponent } from 'react'

import s from './NotFoundView.scss'

class NotFoundView extends PureComponent {
  render() {
    return (
      <div className={s.wrapper}>
        <div className={s.container}>
          <h1>Error 404</h1>
          <p>Sorry, the page you were trying to view does not exist.</p>
        </div>
      </div>
    )
  }
}

export default NotFoundView
