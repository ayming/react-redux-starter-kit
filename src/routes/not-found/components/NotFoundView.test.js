/* eslint-env jest */
import React from 'react'
import renderer from 'react-test-renderer'

import NotFoundView from './NotFoundView'

describe('NotFound', () => {
  test('renders correctly', () => {
    const wrapper = renderer.create(<NotFoundView />).toJSON()

    expect(wrapper).toMatchSnapshot()
  })
})
