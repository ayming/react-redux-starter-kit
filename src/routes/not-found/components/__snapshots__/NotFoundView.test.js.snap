// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NotFound renders correctly 1`] = `
<div
  className="wrapper"
>
  <div
    className="container"
  >
    <h1>
      Error 404
    </h1>
    <p>
      Sorry, the page you were trying to view does not exist.
    </p>
  </div>
</div>
`;
